.github/workflows/check-patch.yml

```diff
@@ -0,0 +1,41 @@
+name: Check patch
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-el8:
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
+
+    steps:
+    - name: Import ovirt-master-snapshot key
+      run: rpm --import https://download.copr.fedorainfracloud.org/results/ovirt/ovirt-master-snapshot/pubkey.gpg
+
```

Comment:
Here you should use 4.4 repo https://download.copr.fedorainfracloud.org/results/ovirt/ovirt-4.4-snapshot/pubkey.gpg